src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/CustomConverterTests/CustomConverterTests.cs

```diff
@@ -161,19 +163,29 @@ public static void VerifyObjectConverterWithPreservedReferences()
             Assert.Equal(true, obj);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
         public static void GetConverterRootsBuiltInConverters()
         {
-            JsonSerializerOptions options = new();
-            RunTest<DateTime>();
-            RunTest<Point_2D>();
-
-            void RunTest<TConverterReturn>()
+            RemoteExecutor.Invoke(static () =>
```

Comment:
this feels a bit like a unit test rather than functional test because it tests an implementation detail. I'll suggest to move it to unit tests.